### PR TITLE
further text, conditions, guidance on unlinkability

### PR DESCRIPTION
**Pull request by npdoty**
explain a bit the threat that unlinkability protects against introduce "inherently linkable"
conditions for when web technology must enable unlinkable presentations what kind of guidance is needed for presentations that still might be linkable

addresses #6

### CODE REVIEW

**Comment by rinchen** [comment]
@npdoty After more thought, my comments on the PING call boil down to best practices. I am not yet informed enough to know whether or not this is in PING's purview or if this is articulated elsewhere. 

- Minimal Disclosure: Only present the necessary claims or properties relevant to the specific context. Avoid including additional information that could lead to linking.

- Context-Specific Identifiers: If any identifiers are required, ensure they are context-specific and do not allow cross-context correlation. For example, use a session-specific token rather than a globally unique identifier.

- User Consent and Transparency: Inform users about the presentation process and any potential risks. Provide clear notices that their claims will not be linked across different contexts unless explicitly authorized.

- Credential Design: When designing credentials, consider unlinkability as a core requirement. Minimize the inclusion of personally identifiable information (PII) or other sensitive data that could be used for tracking.

Thus, my remarks were indeed around a notice of linkability. I was thinking of it mostly from the consent and transparency component above.

e.g.

"The following information may be linked/shared with other agencies to facilitate your ability to use other government services. By pressing submit, you acknowledge and consent to their usage.
 * Drivers License number
 * State of issuance
 * Expiration date"

**Comment by npdoty** [comment]
Discussion from call today also suggested (@jyasskin) that it would be useful to have examples of the implications for different kinds of presentations. For example, revocation status and signature types for not inherently linkable presented claims.

**Comment by npdoty** [comment]
Thanks, @rinchen. I believe your list is a good one, and goes beyond this particular change. Some sections already exist (selective disclosure, for example), and some are in the outline but are incomplete (consent is related to in-context explanations and voluntary presentation). Would you want to [add issues to this list](https://github.com/w3cping/credential-considerations/issues), or suggest some text?

File: credentials-considerations.md [hunk 1]
```diff
@@ -34,14 +34,18 @@ See also: [abstract claims](https://www.w3.org/TR/vc-data-model/#favor-abstract-
...
 
-In some cases the properties themselves are designed for linkability (like a unique government-issued ID number, or a name or address). But many claims, like age or country are not by themselves inherently linkable.
+In some cases the properties themselves are designed for linkability (like a unique government-issued ID number, or a name or address). But many claims, like age or country are not by themselves **inherently linkable**.
+
+For any credential claims that are not inherently linkable from credentials that are designed for usage across multiple origins, presentations should be made unlinkable. Support for other capabilities, such as revocation status, should not make an otherwise unlinkable presentation linkable (see: [Revocation](#revocation-linkability), below). Specifications should define the necessary, testable requirements for presentations to be reasonably unlinkable, and identify all claims that are *inherently linkable*.
```
Comment (jyasskin) [inline credentials-considerations.md]: I got lost in "For any credential claims that are not inherently linkable from credentials that are designed for usage across multiple origins". Is that something like "If a credential is designed for usage across multiple origins, and it contains credential claims that are not inherently linkable, then presentations of those claims should be made unlinkable"?

**Review by jyasskin — APPROVED** [review]
Looks good aside from one confusing sentence. Thanks!

**Pull request merged**